dataset-spec.html

```diff
@@ -273,6 +268,11 @@ <h3>Methods</h3>
     </p>
     <p>Returns an N-Quads string representation of the dataset.</p>
     <p>No normalization is done before, so the results for the same quads may vary based on the Dataset implementation.</p>
+
+    <p>
+      <dfn>union</dfn>
+    </p>
+    <p>Returns a new <a>Dataset</a> that is a concatenation of this dataset and the <code>quads</code> given as an argument.</p>
```

Comment:
"concatenation" may be an oversimplification. If any of the datasets contain equal blank nodes, then the blank nodes should be standardized apart according to the RDF spec.

It may be good to link to the section on [graph merging](https://www.w3.org/TR/rdf11-mt/#shared-blank-nodes-unions-and-merges) in the RDF spec.

Reply:
I haven't changed the wording. So it only shows up as a change, cause it's sorted alphabetically. Let's keep this PR just about the renaming of `merge` to `union`.

Your comments are worth another issue. In the past I've implemented it just as a concatenation and haven't thought about the impact.

Reply:
Good point, opened #22 for this.